### PR TITLE
feature: Add link to multiple reports documentation

### PR DESCRIPTION
**Pull request by franciscodua**

### CODE REVIEW

File: src/main/scala/com/codacy/rules/ReportRules.scala [hunk 1]
```diff
@@ -72,6 +72,14 @@ class ReportRules(coverageServices: => CoverageServices) extends LogSupport {
           sendFilesReportForCommit(files, config, partial = config.partial, commitUUID)
         }
       }
+      if (config.partial) {
+        logger.info(
+          """To complete the reporting process, call the coverage-reporter with a final flag.
+          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
+          | for more information."""
```
Comment (lolgab) [inline src/main/scala/com/codacy/rules/ReportRules.scala:79]: You forgot `.stripMargin` here

Comment (franciscodua) [reply src/main/scala/com/codacy/rules/ReportRules.scala]: True. Already changed it

File: src/main/scala/com/codacy/rules/ReportRules.scala [hunk 2]
```diff
@@ -72,6 +72,14 @@ class ReportRules(coverageServices: => CoverageServices) extends LogSupport {
           sendFilesReportForCommit(files, config, partial = config.partial, commitUUID)
         }
       }
+      if (config.partial) {
+        logger.info(
+          """To complete the reporting process, call the coverage-reporter with a final flag.
+          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
+          | for more information.""".stripMargin
```
Comment (prcr) [inline src/main/scala/com/codacy/rules/ReportRules.scala:79]: Proposing a small tweak to the copy of the message:

```suggestion
          """To complete the reporting process, call coverage-reporter with the final flag.
          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
          | for more information.""".stripMargin
```

Comment (pauloribeiro-codacy) [inline src/main/scala/com/codacy/rules/ReportRules.scala:79]: Suggestion a minor tweak to the copy of the message:

```suggestion
          """To complete the reporting process, call coverage-reporter with the final flag.
          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
          | for more information.""".stripMargin
```

**Pull request merged**